### DISCUSSION TITLE
Pages no longer updates with repo

### DISCUSSION THREAD

**Discussion by WillForan** [community/community#23573] [Other Feature Feedback, Questions, & Ideas]
<p>At some point the <a href="https://labneurocogdevel.github.io/choice-landscape/" rel="noopener nofollow ugc">github page</a> for <a href="https://github.com/LabNeuroCogDevel/choice-landscape" rel="noopener nofollow ugc">my repo</a> no longer updates and is far behind HEAD. I’ve tried switching which branch pages uses and pushing small changes to index pages without any luck.</p>
<p>I’m using <code>curl</code> to avoid any browser cache issues and diffing the output with the current master branch file. It’s been a month since any updates have shown up, but this example has 25 minutes between change and http get.</p>
<p>The github page settings suggest everything is up and running on the correct branch.<br>
The repo uses &lt;50Mb of storage.</p>
<p>I’m at a loss for how to debug<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/2/d2a8f84b3d9d41feb6da8a4af05424ddde2829b7.png" data-download-href="/uploads/short-url/u3AhG2MtHp9TNKV1BwxxxM7XVLp.png?dl=1" title="nogithubpageupdate" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/680742/181077981-76a34d87-ac04-46c7-a68a-df0afd62471d.png" alt="nogithubpageupdate" data-base62-sha1="u3AhG2MtHp9TNKV1BwxxxM7XVLp" width="690" height="303" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/2/d2a8f84b3d9d41feb6da8a4af05424ddde2829b7_2_690x303.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/2/d2a8f84b3d9d41feb6da8a4af05424ddde2829b7_2_1035x454.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/2/d2a8f84b3d9d41feb6da8a4af05424ddde2829b7.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/2/d2a8f84b3d9d41feb6da8a4af05424ddde2829b7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">nogithubpageupdate</span><span class="informations">1084×477 76.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by felipesu19** [comment, marked as answer]
<p>It looks like your <a href="https://github.com/LabNeuroCogDevel/choice-landscape/runs/4886140756?check_suite_focus=true" rel="noopener nofollow ugc">build is failing</a>, with the error</p>
<pre><code class="lang-auto">  Error: fatal: No url found for submodule path 'results/HabitModelVBA' in .gitmodules
  Error: The process '/usr/bin/git' failed with exit code 128
</code></pre>
<p>Hopefully that gives you a starting point in debugging what’s happening</p>

**Comment by WillForan** [comment]
<p>Thanks!  I didn’t know there were logs for page builds.  Should have checkout all the tabs first. That was much easier than I anticipated.</p>
<pre><code class="lang-auto">git rm --cached results/HabitModelVBA/ # &amp;&amp; commit &amp;&amp; push
</code></pre>

**Comment by felipesu19** [comment]
<p>Pages on Actions is a recent release, so its very unfamiliar to pretty much everyone. Glad that helped you get things moving again :slight_smile:</p>
